backend/src/db/migrate/store.js

```diff
@@ -10,6 +12,28 @@ const defaultAdmin = {
   slug: 'admin',
 }
 
+const createCategories = async (session) => {
+  const createCategoriesTxResultPromise = session.writeTransaction(async (txc) => {
+    categories.forEach(({ icon, name }, index) => {
+      const id = `cat${index + 1}`
+      txc.run(
+        `MERGE (c:Category {
+          icon: "${icon}",
+          slug: "${name}",
+          name: "${name}",
+          id: "${id}",
+          createdAt: toString(datetime())
+        })`,
+      )
```

Comment:
What is if there are already categories?

Do we skip this merge?
Have a logging for it?